cmd/lpm/darwin.go

```diff
@@ -32,7 +32,7 @@ func main() {
 		}
 
 		if fi.Mode()&os.ModeSymlink != 0 {
-			link, err := os.Readlink(loc)
+			link, err := filepath.EvalSymlinks(loc)
```

Comment:
I wasn't aware of this method. It looks awesome. I want to test it a little more.